src/System.Private.Xml/src/System/Xml/Dom/XmlDocument.cs

```diff
@@ -22,7 +22,7 @@ public class XmlDocument : XmlNode
         private DomNameTable _domNameTable; // hash table of XmlName
         private XmlLinkedNode _lastChild;
         private XmlNamedNodeMap _entities;
-		// TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
+        // TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
```

Comment:
This one can be deleted as it has no value.

---
In reply to: [110308973](https://github.com/dotnet/corefx/pull/18042#discussion_r110308973) [](ancestors = 110308973)